tests/internal/e2e/test/specs_config.ts

```diff
@@ -119,6 +119,12 @@ export function getSpecGroups(useVcast24: boolean) {
       env: {},
       params: {},
     };
+
```

Comment:
I have a feeling you have a change in `wdio.conf.ts` that you haven't commited based on this

Reply:
Now it's in